compiler-plugin/src/main/kotlin/arrow/meta/quotes/nameddeclaration/stub/typeparameterlistowner/Property.kt

```diff
@@ -0,0 +1,154 @@
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.property
+ *
+ * val Meta.reformatProperty: Plugin
+ *  get() =
+ *   "ReformatProperty" {
+ *    meta(
+ *     property({ true }) { e ->
+ *      Transform.replace(
+ *       replacing = e,
+ *       newDeclaration = """
+ *                          | $modality $visibility $valOrVar $name: $typeReference = $delegateExpressionOrInitializer
+ *                          |   $getter
+ *                          |   $setter
+ *                          """.property
+ *      )
+ *      }
+ *     )
+ *    }
+ * ```
+ *
+ * ## Delegate Properties
+ *
+ * ```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.property
+ *
+ * val Meta.reformatProperty: Plugin
+ *  get() =
+ *   "ReformatProperty" {
+ *    meta(
+ *     property({ true }) { e ->
+ *      Transform.replace(
+ *       replacing = e,
+ *       newDeclaration = """
+ *                          | $modality $visibility $valOrVar $name: $typeReference by $delegate {
+ *                          |   $delegateExpressionOrInitializer
+ *                          | }
+ *                          """.property
+ *      )
+ *      }
+ *     )
+ *    }
+ * ```
+ *
+ * ## Getters
+ *
+ * ```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.property
+ *
+ * val Meta.reformatPropertyGetter: Plugin
+ *  get() =
+ *   "Reformat Property Getter" {
+ *    meta(
+ *     property({ true }) { e ->
+ *      Transform.replace(
+ *       replacing = e,
+ *       newDeclaration = """
+ *                          | $modality $visibility $valOrVar $name: $typeReference =
+ *                          |    get() = $getter
+ *                          """.property
+ *      )
+ *      }
+ *     )
+ *    }
+ * ```
+ *
+ *
+ * ## Setters
+ *
+ * ```kotlin:ank:silent
+ * import arrow.meta.Meta
+ * import arrow.meta.Plugin
+ * import arrow.meta.invoke
+ * import arrow.meta.quotes.Transform
+ * import arrow.meta.quotes.property
+ *
+ * val Meta.reformatPropertySetter: Plugin
+ *  get() =
+ *   "Reformat Property Setter" {
+ *    meta(
+ *     property({ true }) { e ->
+ *      Transform.replace(
+ *       replacing = e,
+ *       newDeclaration = """
+ *                          | $modality $visibility $valOrVar $name: $typeReference =
+ *                          |    set() = $setter
+ *                          """.property
+ *      )
+ *      }
+ *     )
+ *    }
+ * ```
+ */
+class Property(
+  override val value: KtProperty?,
+  val modality: Name? = value?.modalityModifierType()?.value?.let(Name::identifier),
+  val visibility: Name? = value?.visibilityModifierType()?.value?.let(Name::identifier),
+  val `(typeParameters)`: ScopedList<KtTypeParameter> = ScopedList(prefix = "<", value = value?.typeParameters
```

Comment:
The only one left off is `returnType` and `(typeParameters)` here - I'll be looking for examples!